scanner/src/lib/dataproducts/Battery_CarbonFootprint_v0_1.svelte

```diff
@@ -45,51 +45,55 @@
   <div class="divider" />
   <div class="title no-bottom-margin">Manufacturer information</div>
   <div class="subtitle">The details of the battery manufacturer</div>
-  <DataRow label="Name" value={data.manufacturerInformation.name} />
-  <DataRow label="Street name" value={data.manufacturerInformation.streetName} />
-  <DataRow label="Postal code" value={data.manufacturerInformation.postalCode} />
-  <DataRow label="City" value={data.manufacturerInformation.city} />
-  <DataRow label="Country" value={countryListAlpha3[data.manufacturerInformation.country]} />
-  <DataRow label="Website" link column value={data.manufacturerInformation.website} />
-  <DataRow label="Email" column value={data.manufacturerInformation.email} />
+  <DataRow label="Name" value={data.manufacturerInformation?.name} />
+  <DataRow label="Street name" value={data.manufacturerInformation?.streetName} />
+  <DataRow label="Postal code" value={data.manufacturerInformation?.postalCode} />
+  <DataRow label="City" value={data.manufacturerInformation?.city} />
+  <DataRow label="Country" value={countryListAlpha3[data.manufacturerInformation?.country]} />
+  <DataRow label="Website" link column value={data.manufacturerInformation?.website} />
+  <DataRow label="Email" column value={data.manufacturerInformation?.email} />
```

Comment:
Is it possible for `manufacturerInformation` to be null/undefined according to the spec?